tests/14-crypto-entropy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+### Constants
+
+# The default valgrind in Travis-CI (currently Ubuntu xenial, which has
```

Comment:
This is why all we had all those RDRAND failures in the past few days -- the "Illegal instruction" was from old valgrind failing to deal with RDRAND, not from the instruction itself.